components/ServiceForm/index.js

```diff
@@ -0,0 +1,49 @@
+export default function ServiceForm({ onSubmit, formName }) {
```

Comment:
It seems like you do not pass any `formName` from `/pages/create.js` to the `ServiceForm`. Therefore I suggest to delete `formName` here. 
```suggestion
export default function ServiceForm({ onSubmit }) {
```